docs/installation/local.md

```diff
@@ -21,6 +21,10 @@ Running this tool requires installing the nodejs as well as python dependencies.
 ```
 npm install -g bower
 bower install
+pip install virtualenv
+virtualenv venv
+source activate venv # Linux
```

Comment:
This command is not working. The proper way to do it is source `venv/bin/activate` or `. venv/bin/activate`. Can someone please verify this for their own OS, I would like to create and the issue and work on it.

Reply:
Sure go ahead with raising issue and submitting PR.

Reply:
Okay. I'm working on the issue. Thanks.